flux-kube/AUTHORS

```diff
@@ -0,0 +1,3 @@
+Dong Ahn
```

Comment:
FWIW, flux-core removed their authors file with the following motivation:
> AUTHORS file is a pain to remember to maintain, is not
very compatible with our multi-repo "framework" structure, and is
for the most part, redundant with attributions in git commits.

  If you'd like to do the same, here is the change they made to the autotools file to make it not complain: https://github.com/flux-framework/flux-core/commit/4e7cf042195b921c20cb8abfd736e45b67cd9688

Reply:
Great suggestion; will do!